src/main/resources/application.yml

```diff
@@ -8,13 +8,18 @@ spring:
       discovery:
         enabled: true
         service-id: config
-    
+
+    consul:
+      host: localhost
+      port: 8500
+      discovery:
+        prefer-ip-address: true
     
     gateway:
       globalcors:
         corsConfigurations:
           '[/**]':
-            allowedOrigins: "http://localhost:4200"
+            allowedOrigins: "*"
```

Comment:
why is this change made?

Reply:
That was a testing change that I thought I reverted.